snippets/csharp/keywords/keywords.csproj

```diff
@@ -7,7 +7,7 @@
 
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
The project complained about missing "nullable reference types" feature and didn't build in C# 7.3